src/main/java/com/epam/learn/ai/advice/ControllerAdvice.java

```diff
@@ -14,7 +14,7 @@ public class ControllerAdvice {
     @ExceptionHandler
```

Comment:
you could use `    @ExceptionHandler(exception = NotFoundException.class)`
Won't need to use instanceof